tests/x509/limbo/test_limbo.py

```diff
@@ -0,0 +1,125 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+import datetime
+import ipaddress
+import json
+import os
+
+from cryptography import x509
+from cryptography.x509 import load_pem_x509_certificate
+from cryptography.x509.verification import (
+    PolicyBuilder,
+    Store,
+    VerificationError,
+)
+
+LIMBO_UNSUPPORTED_FEATURES = {
+    # NOTE: Path validation is required to reject wildcards on public suffixes,
+    # however this isn't practical and most implementations make no attempt to
+    # comply with this.
+    "pedantic-public-suffix-wildcard",
+    # TODO: We don't support Distinguished Name Constraints yet.
+    "name-constraint-dn",
+    # Our support for custom EKUs is limited, and we (like most impls.) don't
+    # handle all EKU conditions under CABF.
+    "pedantic-webpki-eku",
+    # Similarly: contains tests that fail based on a strict reading of RFC 5280
+    # but are widely ignored by validators.
+    "pedantic-rfc5280",
+    # In rare circumstances, CABF relaxes RFC 5280's prescriptions in
+    # incompatible ways. Our validator always tries (by default) to comply
+    # closer to CABF, so we skip these.
+    "rfc5280-incompatible-with-webpki",
+}
+
+LIMBO_XFAIL_TESTCASES = {
```

Comment:
These seems like they should be skips, not xfails, since they're intentional behavior.

Reply:
9ce06d0c8